### PR TITLE
Add docs links to access-plans article

### PR DESCRIPTION
**Pull request by tzanko-matev**
#### Summary
Add documentation links to access plans article. 

[//]: # (#### Related PRs)
https://github.com/santiment/articles/pull/3

### CODE REVIEW

File: lib/sanbase/clickhouse/metric/helper.ex [hunk 1]
```diff
@@ -9,8 +9,19 @@ defmodule Sanbase.Clickhouse.Metric.Helper do
     Enum.filter(metrics, fn metric -> String.contains?(metric, str) end)
   end
 
+  @doc documentation_ref: """
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/metric/helper.ex]: You can use not heredocs, but simple strings: 
```elixir
@doc documentation_ref: "DOCS access-plans/index.md"
```

Comment (tzanko-matev) [reply lib/sanbase/clickhouse/metric/helper.ex]: Fixed

**Comment by IvanIvanoff** [comment]
@tspenov @tzanko-matev I still think one doc reference per module is enough. Here these refs point to the exact same point in the documentation - this does not enrich the documentation more than having it just once

**Comment by tspenov** [comment]
Yeah maybe one link is enough

**Comment by IvanIvanoff** [comment]
When needed can be reopened and discussed further.

**Pull request closed without merging**